healvis/observatory.py

```diff
@@ -219,9 +249,19 @@ def get_observed_region(self, Nside):
     def vis_calc(self, pcents, tinds, shell, vis_array, Nfin, beam_pol='pI'):
```

Comment:
need a docstring!

Reply:
possibly also more descriptive variable names? `pcents` I get means pointing centres, though that's not _immediately_ clear. But `tinds` I hae no idea, and `Nfin` either.

Reply:
This function is run internally, sent to subprocesses via the `multiprocessing` module. I've made it a private function and added some documentation.